### PR TITLE
namespace fix

### PR DESCRIPTION
**Pull request by thijstriemstra**
fixes https://github.com/rm-hull/luma.core/issues/164

blocked by new luma.core release (also needs to be updated in setup.py)

### CODE REVIEW

**Comment by thijstriemstra** [comment]
ran tests locally and seems fine (python 3.7.3)

File: setup.py [hunk 1]
```diff
@@ -35,7 +34,7 @@ def find_version(*file_paths):
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 test_deps = [
     'mock;python_version<"3.3"',
-    'pytest==4.5',
+    'pytest>=3.1',
```
Comment (rm-hull) [inline setup.py]: pytest 4.6.x breaks coverage

Comment (thijstriemstra) [reply setup.py]: hopefully not anymore ;) works here locally

Comment (thijstriemstra) [reply setup.py]: yea, i'll revert that for now

**Pull request merged**